tasklist/client/README.md

```diff
@@ -65,3 +65,16 @@ You can then destroy the environment by pressing Ctrl+C and running:
 ```sh
 make env-down
 ```
+
+## Generating snapshots for camunda-docs
+
+We use playwright to automate taking snapshots of our application. This utilizes the visual regression
+playwright environment to take snapshots. These are currently 1280x720 pixel images (unless overridden).
+
+To run the snapshot tests:
+```
+yarn start-visual-regression-docker
+# in the docker environment
+yarn start:visual-regression &
```

Comment:
I was thinking about running it separately from the regular visual regression tests, but I suppose it makes no difference

Reply:
It is a separate set of tests I am just reusing the playwright docker container to run them.